README.md

```diff
@@ -69,9 +69,26 @@ $ npm start
 
 3. See data displayed in the admin portal
 
-## Using Demo Data
```

Comment:
Is this stuff about demo data deleted because it's obsolete?

Reply:
Yep, it's obsolete and no longer relevant now :)

Reply:
Yeah, I removed the ability to do this the last time I messed with the webpack config. We hadn't maintained this feature in several years.